README.md

```diff
@@ -485,6 +485,8 @@ Raises
 - `Net::OpenTimeout` on connect timeout
 - `Net::ReadTimeout` on read timeout
 
+Defaults: 60s read timeout.  Connect timeout was infinite till Ruby 2.2, became 60s in Ruby 2.3.
```

Comment:
NEWS: https://github.com/ruby/ruby/blob/v2_3_0/NEWS#user-content-stdlib-compatibility-issues-excluding-feature-bug-fixes,
github: https://github.com/ruby/ruby/commit/52e1c3b0ab41041f7f51a7afc3fce3aab97bc010,
svn: https://svn.ruby-lang.org/cgi-bin/viewvc.cgi/trunk/lib/net/http.rb?view=log#rev52480
linked github as most useful, also includes NEWS addition.

also came across [issue](https://bugs.ruby-lang.org/issues/13396) about inability to set write timeout so linked it.  (obviously guide can't track all issues, but net/http is fundamental and the fact it can hang despite setting all timeouts seems pertinent to guide's goal?)